app/views/app/decision.html.erb

```diff
@@ -155,10 +155,11 @@
           data["signature"] = signature;
         }
         if (res) {
-          postHelper("/offers/<%= @offer[:id] %>/decision/"+status, data, updateStatus);
+          postHelper("/pb/<%= @offer[:mangled] %>/"+status, data, updateStatus);
         }
       }
       function updateStatus(response){
+        fetchPdfData();
```

Comment:
rerender the pdf after accept so they can see that a signature kinda sorta has been rendered into the letter. (and print the signed one)

maybe before accept the sig line should say something like "not yet electrontically signed"